sql-cli/tests/test_configuration.py

```diff
@@ -19,5 +19,4 @@ def test_from_yaml_to_config_without_env():
     config_from_file = config_reference.from_yaml_to_config()
     assert isinstance(config_from_file, Config)
     assert config_from_file.project_dir == config_reference.project_dir
-    assert config_from_file.environment is None
-    assert config_from_file.connections == []
+    assert config_from_file.environment == "default"
```

Comment:
@feluelle, I'd love your thoughts on this change.

My feeling is that it is valid in the use case where we use the Configuration object and we don't care about the environment - but I thought it might be more graceful to have a default value than to have to propagate str/None possibilities through the code, handling scenarios where config would not have an environment.

Reply:
I personally think this makes sense. Default is... well... a default so it might as well be reflected as such in the code.

Reply:
Yes, +1 on explicit `default` as a default environment.